source/_dashboards/statistics-graph.markdown

```diff
@@ -17,7 +17,7 @@ Screenshot of the statistics graph card with none metered entities and `chart_ty
 Screenshot of the statistics graph card with a metered entity and `chart_type` `bar`.
 </p>
 
-Statistics are gathered every 5 minutes for sensors that support it. It will either keep the `min`, `max`, and `mean` of a sensor's value for a specific hour or the `sum` for a metered entity.
+Statistics are gathered every 5 minutes for sensors that support it. It will either keep the `min`[^1], `max`[^1], and `mean`[^1] of a sensor's value for a specific hour or the `sum`[^1] for a metered entity.
```

Comment:
```suggestion
Statistics are gathered every 5 minutes for sensors that support it. It will either keep the `min`, `max`, and `mean` (average) of a sensor's values for a specific hour or the `sum` for a metered entity.
```
counterproposal: how about just adding the term average in brackets? In this way, the information is right where it is needed.

Reply:
I get wat you are saying in my case only mean was confusing, but IMHO we should always be as complete as possible when it comes to documenting. But I leave it up to you how to merge it.

Reply:
I agree with you in that it is good practice to spell out abbreviations. Amended the suggestion to use the `abbr` tag. This will show a pop up.
![image](https://github.com/home-assistant/home-assistant.io/assets/38767475/eab961b8-dbc0-48d9-8716-8525fcc07cde)